src/neuroglancer/segmentation_display_state/frontend.ts

```diff
@@ -414,6 +420,20 @@ function makeRegisterSegmentWidgetEventHandlers(displayState: SegmentationDispla
     stickyChildren[template.visibleIndex].addEventListener('click', visibleCheckboxHandler);
     children[template.filterIndex].addEventListener('click', filterHandler);
     element.addEventListener('action:select-position', selectHandler);
+
+    const trackableRGB = new TrackableRGB(vec3.fromValues(0, 0, 0));
+    trackableRGB.changed.add(() => {
+      const testU = new Uint64(packColor(trackableRGB.value));
+
+      const idString = element.dataset.id!;
+      const id = tempStatedColor;
+      id.tryParseString(idString);
+      displayState.segmentStatedColors.value.delete(id);
+      displayState.segmentStatedColors.value.set(id, testU);
+    });
+
+    // TODO, need to reigster disposer?
+    new ColorWidget(trackableRGB, undefined, children[template.colorWidgetIndex] as HTMLInputElement);
```

Comment:
new ColorWidget is usually added using registerDisposer though I don't see a way to do that in `makeRegisterSegmentWidgetEventHandlers`. Perhaps that is the wrong function to construct it in.

I made changes to ColorWidget so that you could request a template and provide it with an element in the constructor.